gradle/libs.versions.toml

```
@@ -60,7 +60,7 @@ moshi-kotlinReflect = { module = "com.squareup.moshi:moshi-kotlin", version.ref
 okio = { module = "com.squareup.okio:okio", version = "3.8.0" }
 
 tools-common = { module = "com.android.tools:common", version.ref = "androidTools" }
-tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version = "31.2.2" }
+tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version = "31.3.0" }
```

Could this be using the variable or they are not expected to _always_ be released together you think?
```suggestion
tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version.ref = "androidTools" }
```

---

the latter.  the variable used to be shared, but was decoupled to assist the Flamingo bump: https://github.com/cashapp/paparazzi/pull/760